File: LayoutTests/http/tests/security/resources/xorigincss1-allow-star.php
<?php
header("Access-Control-Allow-Origin: *");

$name = 'xorigincss1.css';
$fp = fopen($name, 'rb');
header("Content-Type: text/css");
header("Content-Length: " . filesize($name));

fpassthru($fp);
exit;
?>

File: LayoutTests/http/tests/security/resources/xorigincss1-allow-star.py
#!/usr/bin/env python3

import os
import sys

sys.stdout.write(
    'Access-Control-Allow-Origin: *\r\n'
    'Content-Type: text/css\r\n\r\n'
)

sys.stdout.flush()
with open(os.path.join('/'.join(__file__.split('/')[0:-1]), 'xorigincss1.css'), 'rb') as file:
    sys.stdout.buffer.write(file.read())